github/client.go

```diff
@@ -829,7 +829,8 @@ func (client *Client) simpleApi() (c *simpleClient, err error) {
 }
 
 func (client *Client) apiClient() *simpleClient {
-	httpClient := newHttpClient(os.Getenv("HUB_TEST_HOST"), os.Getenv("HUB_VERBOSE") != "")
+	unixSocket := os.ExpandEnv(client.Host.UnixSocket)
```

Comment:
Why is `ExpandEnv` necessary here?

Reply:
It resolves\expands a path with environment variables, e.g.: `$HOME = /Users/apakulov`
If we have a value `$HOME/go.sock` it should result into `/Users/apakulov/go.sock`